src/jobsautoreport/report.py

```diff
@@ -150,7 +164,7 @@ def get_report(self, from_date: datetime, to_date: datetime) -> Report:
                 periodic_subsystem_and_e2e_jobs
             ),
             top_10_failing_e2e_or_subsystem_periodic_jobs=self._get_top_n_jobs(
-                periodic_subsystem_and_e2e_jobs, self._get_job_success_rate, 10, False
+                periodic_subsystem_and_e2e_jobs, self._get_job_success_rate, 10, True
```

Comment:
let's do usage of named-keywords. without it it's very hard to understand what each value refers to. E.g.:
```
top_10_failing_e2e_or_subsystem_periodic_jobs=self._get_top_n_jobs(
    jobs=periodic_subsystem_and_e2e_jobs,
    computation_func=self._get_job_success_rate,
    n=10,
    is_reverse=True,
)
```